flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -584,11 +587,38 @@ public void received(AllocationProfile response) {
     });
   }
 
+  protected void gcNow(FlutterStudioMonitorStageView view) {
+    FlutterStudioProfilers profilers = getStage().getStudioProfilers();
+    VmService vmService = profilers.getApp().getVmService();
+
+    final CompletableFuture<AllocationProfile> future = new CompletableFuture<AllocationProfile>();
+
+    vmService.getAllocationProfile(isolateId, "full", null, new AllocationProfileConsumer() {
+      @Override
+      public void onError(RPCError error) {
+        LOG.error("Allocation Profile during gcNow - " + error.getDetails());
+        future.completeExceptionally(new RuntimeException(error.toString()));
+      }
+      @Override
+      public void received(AllocationProfile response) {
+        // TODO(terry): Add GC Icon to overlay - should happen in FlutterAllMemoryData however handleGCEvent is never called.
+        view.getStage().recordGC();
+      }
+    });
+  }
+
+  // TODO(terry): Remove for testing only.
```

Comment:
remove

Reply:
Comment was not accurate should read:
  // TODO(terry): Need to remove when we 3.2 of AS isn't supported only post 3.2 have this png.